TypeCobol/Compiler/Nodes/Procedure.cs

```diff
@@ -153,11 +153,7 @@ public FunctionDeclaration(FunctionDeclarationHeader header) : base(header)
 
         public override bool VisitNode(IASTVisitor astVisitor)
         {
-            return astVisitor.Visit(this) && this.ContinueVisitToChildren(astVisitor, Profile.InputParameters)
```

Comment:
Why remove these lines ?

Reply:
To avoid duplicate visit of parameters. Parameters are created as nodes and now properly referenced as children of `FunctionDeclaration` so they are visited as part of the traversal of the tree and the parent don't need to call `ContinueVisitToChildren`. The `VisitNode` method is designed to visit the elements of the node itself not children nodes.